app/views/menu/menu.html.erb

```diff
@@ -1,156 +1,140 @@
-          <tr>
-    <th>Dish Name</th>
-    <% if current_user %> <%# for staff %>
-        <th>Preset Meals</th>
-        <th>HC Prep</th>
-        <th>HC Leftover</th> 
-        <th>BB Prep</th>
-        <th>BB Leftover</th>
-        <th>Buckley Prep</th>
-        <th>Buckley Leftover</th>
-    <% end %>
-  </tr>
-              <tbody>
-                    <% @daily_dishes_menus.each do |dish_menu|%>
-                    <% dish = Dish.find(dish_menu.dish_id) %>
-                     <%# print the table %>
-                    <tr>
-                        <td><%= dish.name %></td>
-                         <% if current_user %> <%# for staff %>
-                            <td><%= 
-                            if (dish.star == 0) 
-                            ""
-                            elsif (dish.star == 1) 
-                            "Dinner"
-                            elsif (dish.star == 2) 
-                            "Supper"
-                            else 
-                            "D + S"
-                            end %></td>
-                            <td><%= dish_menu.hc_prep %></td>
-                            <td><%= dish_menu.hc_leftover %></td> 
-                            <td><%= dish_menu.bb_prep %></td>
-                            <td><%= dish_menu.bb_leftover %></td> 
-                            <td><%= dish_menu.buckley_prep %></td>
-                            <td><%= dish_menu.buckley_leftover %></td> 
-                        <% end %> 
+                    <div>
                         <% if current_user && current_user.admin %>
-                        <td><a  href="/">TEMP remove dish</a></td>
-                        <td><a  href="/">PERM remove dish</a></td>
-                        <script>
-                            if (window.location.href.split("?").length > 1) {
-                                date = window.location.href.split("?")[1].split("=")[1];
-                            } else {
-                                today = new Date();
-                                date = today.getDate() + "/" + (today.getMonth() + 1) + "/" + today.toDateString().split(" ")[3];
-                            }
-                            document.getElementById("del_dinner_temp_<%= dish %>").href = "/menu/edit?date=" + date + "&test=1&act=remove&type=Dinner&permanent=false&dish=<%= dish %>"
-                            document.getElementById("del_dinner_perm_<%= dish %>").href = "/menu/edit?date=" + date + "&test=1&act=remove&type=Dinner&permanent=true&dish=<%= dish %>"
-                        </script>
+                            <!--TODO: Select Search Dropdown-->
+                            <input type="text" class="form-control" id="dinner_field" onkeydown="url()">
+                            <br>
+                            <button class="btn btn-primary">Add Dish to
+                            Menu</button></a> <%# TODO: this button has not been
```

Comment:
the newline you added in the middle of the button text is causing AuthTest line 19 to fail